File: src/main/java/com/paragon/client/systems/module/impl/client/rotation/RotationPriority.java
package com.paragon.client.systems.module.impl.client.rotation;

/**
 * @author Wolfsurge
 * @since 23/03/22
 */
public enum RotationPriority {
    HIGHEST(2),
    HIGH(1),
    NORMAL(0),
    LOW(-1),
    LOWEST(-2);

    private int priority = 0;

    RotationPriority(int priority) {
        this.priority = priority;
    }

    public int getPriority() {
        return priority;
    }
}

File: src/main/java/com/paragon/client/managers/rotation/RotationPriority.kt
package com.paragon.client.managers.rotation

/**
 * @author Surge
 * @since 23/03/22
 */
enum class RotationPriority(priority: Int) {
    HIGHEST(2), HIGH(1), NORMAL(0), LOW(-1), LOWEST(-2);

    var priority = 0

    init {
        this.priority = priority
    }
}